test/ca_bundle/ca_bundle_test.go

```diff
@@ -49,14 +49,14 @@ func TestBundle(t *testing.T) {
 		//before test run
 		log.Printf("resource file location %s find target %t", parameter.dataInput, parameter.findTarget)
 		t.Run(fmt.Sprintf("resource file location %s find target %t", parameter.dataInput, parameter.findTarget), func(t *testing.T) {
-			test.ReplaceLocalStackHostName(parameter.dataInput + configJSON)
-			test.CopyFile(parameter.dataInput+configJSON, configOutputPath)
-			test.CopyFile(parameter.dataInput+commonConfigTOML, commonConfigOutputPath)
-			test.StartAgent(configOutputPath, true)
+			util.ReplaceLocalStackHostName(parameter.dataInput + configJSON)
+			util.CopyFile(parameter.dataInput+configJSON, configOutputPath)
```

Comment:
Hmm !!! Would it be better with agent with related functions and /aws package for all functionalities regarding cwl and cwm since I would expect more service such as IMDS or SSM soon.

Reply:
I don't think I understood. What are you suggesting?

Reply:
So /agent with related functions, /aws for cwl,cwm (and extra services such as ssm)